src/clients/cassandra/CassandraConnector.js

```diff
@@ -93,6 +93,30 @@ function executeQuery(query, params, options) {
 }
 
 /**
+ * @param {string} query
+ * @param {string[]} params
+ * @param {int} fetchSize
+ * @param {string} pageState
+ * @returns {Promise.<object[]>}
+ */
+function executeQueryWithPageState(query, params, pageState, fetchSize) {
+  return new Promise((resolve, reject) => {
+      const DEFAULT_FETCH = 15;
+
+      let options = {
+        fetchSize: fetchSize || DEFAULT_FETCH
+      };
+
+      if(pageState)
```

Comment:
Nit: we use space before opening parens and curly braces around statements, so:

```
if (pageState) {
  options.pageState = pageState
}
```

Added question: would this be different from always setting the pageState? If it's undefined then the downstream library should just be able to handle that just as if it wasn't set:

```
options = {
  pageState: pageState || undefined
}
```